azurerm/data_source_batch_pool.go

```diff
@@ -110,6 +110,20 @@ func dataSourceArmBatchPool() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"container_configuration": {
```

Comment:
This is a data source so this should all be computed, not optional and needs no validatino or max items.